src/sync.d

```diff
@@ -4012,9 +4021,15 @@ class SyncEngine {
 		if (isDir(path)) {
 			if (!appConfig.surpressLoggingOutput) {
 				if (!cleanupLocalFiles) {
-					addProcessingLogHeaderEntry("Scanning the local file system '" ~ logPath ~ "' for new data to upload");
+					addProcessingLogHeaderEntry(
+						"Scanning the local file system '" ~ logPath ~ "' for new data to upload",
```

Comment:
Please dont split this across multiple lines

Please fix for all of these

Reply:
Fixed with 3c2a83fcde35b8ac212e9f083ca14413ad355c58